bonus_guides/Y_generate_project_docs.md

```diff
@@ -0,0 +1,25 @@
+The correct invocation of helper modules and functions can be intimidating because these are generated dynamically (for example, when creating a new project or adding a new `resource`) and they are not documented explicitly (e.g., `MyApp.ErrorHelpers.error_tag`) or the documentation does not cover all examples (e.g., `MyApp.Router.Helpers.*_path` in `Phoenix.Router`).
```

Comment:
So this as an opening statement is a bit confusing. Because it discusses the confusion around generated helpers but doesn't describe how generating Docs solves it. 

Start with a discussion about generating docs for your project and lead into how the process of generating the docs also illuminates the helper functions.